src/utils/SortScenarioListUtils.js

```diff
@@ -27,3 +27,15 @@ export const sortScenarioList = (scenarioList) => {
 
   return sortedList;
 };
+
+export const getFirstScenarioMaster = (scenarioList) => {
+  if (scenarioList.length > 0) {
+    const scenarioMasterList = scenarioList.filter((scenario) => scenario.parentId === null);
+
+    return scenarioMasterList.sort((scenarioA, scenarioB) =>
+      scenarioB.name.toUpperCase() > scenarioA.name.toUpperCase() ? -1 : 1
```

Comment:
In the function just above this one, we are using
`a.name.toUpperCase() < b.name.toUpperCase()` instead of `scenarioB.name.toUpperCase() > scenarioA.name.toUpperCase` to sort the scenarios alphabetically (note that the comparison operator is different).
Could you factorize this in a separate function that we can reuse please ? (this would prevent errors such as different comparison operators in the sorting function)